src/ui/FormWidgets/TextInput.ts

```diff
@@ -33,13 +35,20 @@ export interface ITextInputOptions {
    * A custom aria-label attribute to add to the `TextInput`'s HTML element.
    */
   ariaLabel?: string;
+  /**
+   * The name or svg source of an icon to display inside the input at its beginning.
+   *
+   * **Default:** `null`
+   */
+  prefixingIcon?: string;
 }
 
 const defaultOptions: ITextInputOptions = {
   usePlaceholder: false,
   className: 'coveo-input',
   triggerOnChangeAsYouType: false,
-  isRequired: true
+  isRequired: true,
+  prefixingIcon: null
```

Comment:
Would an empty string work for the default value? Or is `null` needed?

Reply:
or just not defining it since it's optional

Reply:
Good point about using an empty string or outright removing it. I left this over after changing the type from a type guard to a string. I'm going to remove this.